dockerfiles/run/guice/cassandra-rabbitmq-ldap/destination/conf/blob.properties

```diff
@@ -1,17 +1,118 @@
+# ============================================= BlobStore Implementation ==================================
+# Choose your BlobStore implementation
+# Compulsory, allowed values are: cassandra, objectstorage, union
+# union is using both objectstorage as the current BlobStore & cassandra as the legacy BlobStore
 implementation=objectstorage
 
-# Configuration for swift BlobStore
+# ============================================== ObjectStorage ============================================
+
+# ========================================= ObjectStorage Encryption ======================================
+# Codec for blob contents
+# Compulsory, allowed values are: DEFAULT, AES256
+# DEFAULT: no encryption
+# AES256: AES-256 encryption
 objectstorage.payload.codec=DEFAULT
+
+# ===================================== ObjectStorage AES256 Encryption ====================================
+# Compulsory, salt value in string for the encryption
+# objectstorage.aes256.hexsalt=salt
+
+# Compulsory, password for the encryption
+# objectstorage.aes256.password=password
+
+# ========================================= ObjectStorage providers ========================================
+# Choosing blob storage service
+# Compulsory, Allowed values are: swift, aws-s3
 objectstorage.provider=swift
-objectstorage.namespace=james
+
+# ========================================= ObjectStorage Buckets ==========================================
+# bucket names prefix
+# Optional, default no prefix
+# objectstorage.bucketPrefix=prod-
+
+# Default bucket name
+# Optional, default is bucketPrefix + `default`
+# objectstorage.namespace=james
+
+# ========================================= ObjectStorage on Swift =========================================
+# Specify the geography region which the BlobStore will connect to a in Swift ObjectStorage
+# Optional, default is the default region
+# objectstorage.swift.region=HKG
+
+# Specify the authentication mechanism of Swift
+# Compulsory, allowed values are: tmpauth, keystone2, keystone3
 objectstorage.swift.authapi=tmpauth
+
+# Compulsory, authentication endpoint
 objectstorage.swift.endpoint=http://swift:8080/auth/v1.0
+
+# Compulsory, authentication credential
 objectstorage.swift.credentials=testing
```

Comment:
what's the format?

Reply:
oh it's the `Secret Access Key` in jcloud, is that okay to you?

Reply:
what I mean is that credentials is very often several values (login/password being the most used).
Requiring a single entry looks weird so maybe it refers to a swift specific concept but it may be cool to explain it.